server/controllers/contacts.js

```diff
@@ -302,13 +302,13 @@ router.post('', function (req, res) {
 
 });
 
-router.put('', function (req, res) {
+router.put('/:group_id/:name', function (req, res) {
```

Comment:
Make sure on POST method there is no group_id+name in the body. Otherwise it will be and Update. 
We should NOT be able to do updates with POST method